apps/src/dance/lab2/views/dance-view.module.scss

```diff
@@ -0,0 +1,40 @@
+@import 'color.scss';
+@import './constants.scss';
+
+.danceLab {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  gap: 2px;
+}
+
+.visualizationArea {
+  flex: 0;
+  height: 100%;
+  background-color: $neutral_dark10;
+}
+
+.visualizationColumn {
+  margin: 2px 5px;
+}
+
+.visualization {
```

Comment:
This sort of styling is managed by each lab2 component individually?

Reply:
Yeah, my thinking is that for now we can keep these styles within Dance, but as we port over other labs with similar layouts, we can look at sharing common styles as needed. Because Music Lab and AI Chat have very different layouts, there's not much to share right now, but I imagine we'll be able to share more with labs that have a more "traditional" layout (visualization on the side, instructions above workspace).